includes/templates/template_default/templates/tpl_modules_create_account.php

```diff
@@ -127,7 +127,7 @@
 ?>
 <br class="clearBoth" />
 <label class="inputLabel" for="fax"><?php echo ENTRY_FAX_NUMBER; ?></label>
-<?php echo zen_draw_input_field('fax', '', 'id="fax"', 'tel') . (zen_not_null(ENTRY_FAX_NUMBER_TEXT) ? '<span class="alert">' . ENTRY_FAX_NUMBER_TEXT . '</span>': ''); ?>
+<?php echo zen_draw_input_field('fax', '', 'id="fax"', 'fax') . (zen_not_null(ENTRY_FAX_NUMBER_TEXT) ? '<span class="alert">' . ENTRY_FAX_NUMBER_TEXT . '</span>': ''); ?>

```

Comment:
Got it – thanks!  I need to resubmit the Referrals Mod I just submitted, because I changed it to ‘fax’  Please disregard the other submission.

Judy Gunderson



From: Chris Brown 
Sent: Sunday, January 3, 2016 3:03 PM
To: zencart/zc-v1-series 
Cc: judygunderson 
Subject: Re: [zc-v1-series] Update tpl_modules_create_account.php (#751)

In includes/templates/template_default/templates/tpl_modules_create_account.php:

> @@ -127,7 +127,7 @@
>  ?>
>  <br class="clearBoth" />
>  <label class="inputLabel" for="fax"><?php echo ENTRY_FAX_NUMBER; ?></label>
> -<?php echo zen_draw_input_field('fax', '', 'id="fax"', 'tel') . (zen_not_null(ENTRY_FAX_NUMBER_TEXT) ? '<span class="alert">' . ENTRY_FAX_NUMBER_TEXT . '</span>': ''); ?>
> +<?php echo zen_draw_input_field('fax', '', 'id="fax"', 'fax') . (zen_not_null(ENTRY_FAX_NUMBER_TEXT) ? '<span class="alert">' . ENTRY_FAX_NUMBER_TEXT . '</span>': ''); ?>
No, it's supposed to be tel there.
The HTML spec recognizes <input type='tel'> but doesn't recognize <input type='fax'>

—
Reply to this email directly or view it on GitHub.